app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
Em tìm hiểu cho chị về `present?` nhé

so sánh sự khác nhau giữa present? và nil? nào

Reply:
nil? dùng để check xem có nil không, chỉ trả về true với trường hợp là nil. Còn present? kiểm tra sự tồn tại là hàm ngược với blank?, blank? trả về true trong cả nil, false và empty, tương tự thì present đã bao gồm cả not nil, not empty, not false nên dùng present? check được phạm vi lớn hơn khi kiểm tra tính tồn tại. Trong trường hợp trên nên dùng present?